### PR TITLE
Added grpc-protos & alts to upb-generated

### PR DESCRIPTION
**Pull request by veblush**
For gRPC C Core to use upb for everything, all protos used by nanopb are going to generate corresponding upb files.

### CODE REVIEW

File: BUILD [hunk 1]
```diff
@@ -2317,6 +2317,81 @@ grpc_cc_library(
     tags = ["no_windows"],
 )
 
+grpc_cc_library(
```
Comment (nicolasnoble) [inline BUILD]: No, you need to use the new grpc_upb_proto_library for this.

Comment (veblush) [reply BUILD]: Done

File: tools/codegen/core/gen_upb_api.sh [hunk 2]
```diff
@@ -36,6 +43,11 @@ proto_files=( \
   "google/protobuf/wrappers.proto" \
   "google/rpc/status.proto" \
   "gogoproto/gogo.proto" \
+  "grpc/health/v1/health.proto" \
+  "grpc/lb/v1/load_balancer.proto" \
+  "src/core/tsi/alts/handshaker/proto/handshaker.proto" \
```
Comment (nicolasnoble) [inline tools/codegen/core/gen_upb_api.sh]: Maybe we should alphabetize this list. I didn't realize it wasn't.

Comment (veblush) [reply tools/codegen/core/gen_upb_api.sh]: Done :)

**Pull request merged**